client/src/components/related/related-components/RelatedProduct.jsx

```diff
@@ -26,9 +26,10 @@ export default function RelatedProduct({ feature, relProd }) {
         setRelProdMeta(data[2].data);
       }))
       .catch((err) => console.error(err));
-  }, [feature, relProd.id]);
+    // Avoid infinity loop
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I am learning more about this rule and will try to see what we can do to avoid breaking it, but for now this is fine.